app/javascript/components/ForgotPassword.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Axios from 'axios';
+
+export default class ForgotPasswordForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { user: { email: '', redirect_url: 'http://localhost:3000/reset_password' } };
```

Comment:
do you think localhost will be working on staging?

Reply:
I think on staging will be